packages/cli/src/lib/snapshot/vaultEntriesFunctions/transferFromOrganization.ts

```diff
@@ -0,0 +1,123 @@
+import {VaultFetchStrategy, VaultEntryModel} from '@coveord/platform-client';
+import {CliUx} from '@oclif/core';
+import {bold, green, red} from 'chalk';
+import dedent from 'ts-dedent';
+import {SnapshotMissingVaultEntriesFromOriginError} from '../../errors/vaultErrors';
+import {AuthenticatedClient} from '../../platform/authenticatedClient';
+import {Project} from '../../project/project';
+import type {VaultEntryAttributes} from '../snapshotReporter';
+import type {VaultTransferFunctionsParam} from './interfaces';
+
+export async function tryTransferFromOrganization({
+  reporter,
+  snapshot,
+  projectPath,
+}: VaultTransferFunctionsParam) {
+  if (!projectPath) {
+    return false;
+  }
+
+  const originOrgId = new Project(projectPath).getResourceManifest()?.orgId;
+  if (!originOrgId) {
+    return false;
```

Comment:
A message here could be useful... Otherwise it will jump to the next prompt without explaining to the user why.
`CliUx.ux.warn('Unable to locate vault entries from origin org...')` (something like that)